From 1b8012ee8389265ae4539b0573263f2f6f8ae632 Mon Sep 17 00:00:00 2001
Message-ID: <1b8012ee8389265ae4539b0573263f2f6f8ae632.1779975966.git.bertrand.marquis@arm.com>
In-Reply-To: <55d56a174a0f04520d4a1558630bb0b739074b13.1779975966.git.bertrand.marquis@arm.com>
References: <55d56a174a0f04520d4a1558630bb0b739074b13.1779975966.git.bertrand.marquis@arm.com>
From: Bertrand Marquis <bertrand.marquis@arm.com>
Date: Thu, 16 Apr 2026 19:30:47 +0200
Subject: [PATCH 29/30] hw/cxl: add cxl_fmws_get_all_sorted() stub for host
 stubs build

Add the host-stubs build helper needed in hw/cxl.

Keep the patch isolated as a late fix on top of the bridge series.

Signed-off-by: Bertrand Marquis <bertrand.marquis@arm.com>
---
 hw/cxl/cxl-host-stubs.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/hw/cxl/cxl-host-stubs.c b/hw/cxl/cxl-host-stubs.c
index c015baac8138..5bb63e9babb4 100644
--- a/hw/cxl/cxl-host-stubs.c
+++ b/hw/cxl/cxl-host-stubs.c
@@ -16,5 +16,9 @@ hwaddr cxl_fmws_set_memmap(hwaddr base, hwaddr max_addr)
     return base;
 };
 void cxl_fmws_update_mmio(void) {};
+GSList *cxl_fmws_get_all_sorted(void)
+{
+    return NULL;
+}
 
 const MemoryRegionOps cfmws_ops;
-- 
2.53.0

